mne/channels/_standard_montage_utils.py

```diff
@@ -178,7 +184,23 @@ def _read_csd(fname, head_size):
     if head_size is not None:
         pos *= head_size / np.median(np.linalg.norm(pos, axis=1))
 
-    return make_dig_montage(ch_pos=OrderedDict(zip(ch_names, pos)))
+    return make_dig_montage(ch_pos=_dedup_dict(ch_names, pos))
+
+
+def _dedup_dict(ch_names, pos):
```

Comment:
this function is not actually deduplicating but just warning if there are dupes. I find name misleading.

Please rename _dedup_dict to _check_dupes_odict to also say you get an ordered dict out.